[2021-11-29 10:00:43,153] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-11-29T09:59:29.346844+00:00 [queued]>
[2021-11-29 10:00:43,315] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-11-29T09:59:29.346844+00:00 [queued]>
[2021-11-29 10:00:43,345] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-11-29 10:00:43,346] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-11-29 10:00:43,347] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-11-29 10:00:43,751] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_4> on 2021-11-29T09:59:29.346844+00:00
[2021-11-29 10:00:43,870] {standard_task_runner.py:52} INFO - Started process 73103 to run task
[2021-11-29 10:00:43,990] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_4', '2021-11-29T09:59:29.346844+00:00', '--job-id', '15', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpsudbima2', '--error-file', '/tmp/tmpwelpwcqo']
[2021-11-29 10:00:44,053] {standard_task_runner.py:77} INFO - Job 15: Subtask task_4
[2021-11-29 10:00:55,220] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_4 2021-11-29T09:59:29.346844+00:00 [running]> on host airflowvm
[2021-11-29 10:00:55,874] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2021-11-29T09:59:29.346844+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-11-29T09:59:29.346844+00:00
[2021-11-29 10:00:55,876] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-11-29 10:00:55,883] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-11-29 10:00:55,918] {subprocess.py:75} INFO - Output:
[2021-11-29 10:00:58,979] {subprocess.py:83} INFO - Command exited with return code 0
[2021-11-29 10:00:59,639] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_4, execution_date=20211129T095929, start_date=20211129T100043, end_date=20211129T100059
[2021-11-29 10:00:59,918] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-11-29 10:00:59,962] {local_task_job.py:151} INFO - Task exited with return code 0
